### PR TITLE
Update 05-psyplot-simple_case.md

### PR DESCRIPTION
**Pull request by hermanfugl**

### CODE REVIEW

File: _episodes/05-psyplot-simple_case.md [hunk 1]
```diff
@@ -197,35 +197,34 @@ Start a new **python3** notebook on your JupyterHub and type the following comma
...
+ds = xr.open_dataset(filename, decode_times=False)
+
+# and plot surface temperature
+ds.TS.plot()
 ~~~
 {: .language-python}
 
 
-<img src="../fig/TS_F2000_CAM5_T31T31_test-0009-01.png">
+<img src="../fig/test_0009-01">
```
Comment (annefou) [inline _episodes/05-psyplot-simple_case.md]: It should be test_0009-01.png

Could you fix it (select your branch on github and edit your branch directly). Do not make another pull request. 

Thanks.

**Comment by annefou** [comment]
@hermanfugl Can you try to fix this? Thanks.

File: _episodes/05-psyplot-simple_case.md (hunk 1 above)
Comment (hermanfugl) [reply _episodes/05-psyplot-simple_case.md]: I think I fixed it now
